### PR TITLE
added new logo

### PR DESCRIPTION
**Pull request by ruudvdent**
new logo

### CODE REVIEW

**Comment by ruudvdent** [comment]
I suggest to close #104 with this PR until we see any reason to create a new logo later

**Review by Imme1992 — APPROVED** [review]

File: docs/_static/WAM_logo_v3.png:Zone.Identifier [hunk 1]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (Peter9192) [inline docs/_static/WAM_logo_v3.png:Zone.Identifier:1]: This file is related to WSL I believe, you can remove it. I guess it was added to git accidentily.

File: docs/conf.py [hunk 2]
```diff
@@ -10,7 +10,7 @@
 
 myst_enable_extensions = ["dollarmath"]
 
-html_logo = "_static/logo.png"
+html_logo = "_static/WAM_logo_v3.png"
```
Comment (Peter9192) [inline docs/conf.py:13]: Old logo's are also available in git history, so we might as well keep the version out of the logo.

**Review by Peter9192 — COMMENTED** [review]
Bit late to the party, but I noticed a few things that could be considered next time.

**Pull request merged**